packages/app/src/elements/import-dialog.ts

```diff
@@ -281,9 +271,17 @@ export class ImportDialog extends Dialog<File, void> {
                 this._items = await imp.asLastPass(file);
                 break;
             case imp.CSV.value:
-                const result = await imp.asCSV(file, this._itemColumns, this._csvHasColumnsOnFirstRowButton.active);
+                const result = await imp.asCSV(
+                    file,
+                    resetCSVColumns ? [] : this._itemColumns,
+                    this._csvHasColumnsOnFirstRowButton.active
+                );
                 this._items = result.items;
                 this._itemColumns = result.itemColumns;
+                console.log("itemcolumns", this._itemColumns);
+                await this.updateComplete;
```

Comment:
This is what you were missing with regards to the select value not taking hold. Basically the problem was that the `option` elements associated with each column didn't have time to be rendered, which is why the value wasn't properly accepted. This allows the select element to update properly before setting the value.

Reply:
Thanks. I didn't see it in the documentation, but I found it now!